tests/test_apiclient.py

```diff
@@ -591,6 +591,22 @@ def test_update_service_status(self, data_client, rmock):
         assert result == {"services": "result"}
         assert rmock.called
 
+    def test_revert_service(self, data_client, rmock):
+        rmock.post(
+            "http://baseurl/services/123/revert",
+            json={},
+            status_code=200,
+        )
+
+        data_client.revert_service(123, 314159, "jollypoldy@example.com")
+
+        assert tuple(req.json() for req in rmock.request_history) == (
+            {
+                "archivedServiceId": 314159,
+                "updated_by": "jollypoldy@example.com",
+            },
+        )
+
```

Comment:
The other tests seem to be prefer just checking `request_history[0]` directly, or `last_request`. I think this implementation confuses things by making you think about iterables when we know we're only interested in one call.

Reply:
The other way doing it is fine as long as you also check `len(request_history) == 1`.